package.json

```diff
@@ -47,6 +47,10 @@
     "prettier": "^1.14.3",
     "rimraf": "^2.6.2"
   },
+  "peerDependencies": {
+    "eslint": "^4.0.0 || ^5.0.0",
```

Comment:
can you remove `eslint` from dependencies with this change? Also, remove https://github.com/jest-community/jest-runner-eslint/blob/master/src/utils/getLocalESLint.js?

Reply:
Right, I just moved it to `devDeps`...

I guess https://github.com/jest-community/jest-runner-eslint/blob/master/src/utils/getLocalESLint.js now just becomes a regular `require('eslint')` right?

Reply:
yeah, that's the theory!

Reply:
It can be both a dep and a peer dep, or a peer and a dev - in either case it will do what you expect. The difference, though, is that if it’s a dep + peer, then it might not be a breaking change to add the peer - if it’s a peer+dev, then it definitely is breaking.

Reply:
@ljharb thanks, you are great!

Reply:
when it's in `dependencies` now, we will always require our own, and never the user's. Which means that this now moves from eslint 4 to eslint 5, which is a breaking change either way.

I suppose we could roll back to eslint v4, but we really should allow people to use eslint v5 with this plugin

Reply:
Not when it's also in peer dependencies - if the user specifies eslint 4, then we won't install v5, we'll prefer their v4.